src/Components/ElementsPanel/ViewerElementsPanel.types.ts

```diff
@@ -22,6 +22,7 @@ export interface IViewerElementsPanelProps {
 export interface IViewerElementsPanelListProps {
     isLoading: boolean;
     panelItems: Array<IViewerElementsPanelItem>;
+    isModal?: boolean; // TODO: Make this required when deprecated Elements list is removed
```

Comment:
can we add a comment to this prop on when it should be true? I don't quite know just from the name. Is it that the list is in a modal or that the contents should be a modal? 🤔